provisioned-resource.go

```diff
@@ -29,10 +30,10 @@ func (r *ProvisionedResource) GiveMe(target uint32) {
 }
 
 func (r *ProvisionedResource) Start(ctx context.Context) error {
-	// nothing to do
+	r.emit("capacity", int(r.maxCapacity), nil)
```

Comment:
Minor suggestion for keeping this call and any future MaxCapacity() implementation details in sync.
```suggestion
	r.emit("capacity", r.MaxCapacity(), nil)
```

Reply:
Fair enough, I could make that change.